### PR TITLE
Get annotation for a single document

### PR DESCRIPTION
**Pull request by dragon-dxw**

### CODE REVIEW

**Comment by dragon-dxw** [comment]
Alternatively we could go down a route like:
```xquery
xquery version "1.0-ml";

import module namespace dls="http://marklogic.com/xdmp/dls" at "/MarkLogic/dls.xqy";

let $uri := "/eat/2022/1.xml"
let $versions := ($uri, dls:document-version-uris($uri))

for $version in $versions
    return <version><uri>{$version}</uri><annotation>{xdmp:document-properties($version)//dls:annotation/text()}</annotation></version>
```    
which returns a list of versions (including the current judgment) and their annotations.

File: smoketest/smoketest-setup.xqy [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+          <uk:number>1234</uk:number>
+          <uk:cite>[2023] EAT 1234</uk:cite>
+          <uk:parser>0.13.3</uk:parser>
+          <uk:hash>1234</uk:hash>
+        </proprietary>
+    </meta>
+  </judgment>
+</akomaNtoso>
+
+(: return dls:document-delete($URI, fn:false(), fn:false()) :)
```
Comment (anthonyhashemi) [inline smoketest/smoketest-setup.xqy:36]: can this commented line be deleted?

File: tests/client/test_get_annotation.py [hunk 2]
```diff
@@ -0,0 +1,27 @@
+import unittest
+from unittest.mock import patch
+
+from caselawclient.Client import MarklogicApiClient
+
+
+class TestGetSetAnnotation(unittest.TestCase):
```
Comment (anthonyhashemi) [inline tests/client/test_get_annotation.py:7]:
```suggestion
class TestGetAnnotation(unittest.TestCase):
```

Comment (anthonyhashemi) [reply tests/client/test_get_annotation.py:7]: or are we adding a test for setting annotations in this class also?

**Review by anthonyhashemi — APPROVED** [review]
Couple of minor tidy up comments in the tests but looks good 👍

**Pull request merged**